src/js/components/tasks/table.tsx

```diff
@@ -157,18 +155,18 @@ const AssigneeTableCell = ({
   ...props
 }: TableCellProps & {
   type: OrgTypes;
-  projectId: string;
   epicUsers?: GitHubUser[];
-  githubUsers: GitHubUser[];
+  githubUsers?: GitHubUser[];
   currentUser?: User;
   canAssign: boolean;
   isRefreshingUsers: boolean;
-  assignUserAction: AssignUserAction;
+  assignUserAction?: AssignUserAction | undefined;
   children?: string | null;
 }) => {
   const { t } = useTranslation();
+
   const assignedUser = useSelector((state: AppState) =>
-    selectProjectCollaborator(state, projectId, children),
+    selectProjectCollaborator(state, item?.root_project, children),
```

Comment:
We can't really use this anymore, because we're not guaranteed that the relevant Project exists in the Redux store (because Projects are paginated). Again, I think this would be solved if we nest `assigned_qa` and `assigned_dev` in each Task?